src/OpenTelemetry.Instrumentation.Runtime/description.md

```diff
@@ -0,0 +1,175 @@
+
+  - [GCGenerationInfo.FragmentationAfterBytes Property](https://docs.microsoft.com/dotnet/api/system.gcgenerationinfo.fragmentationafterbytes#system-gcgenerationinfo-fragmentationafterbytes)
+  Gets the fragmentation in bytes on exit from the reported collection.
+
+### Additional GC metrics only available for NET6_0_OR_GREATER
+
+| Name                                    | Description        | Units | Instrument Type | Value Type | Attribute Key(s) | Attribute Values           |
+|-----------------------------------------|--------------------|-------|-----------------|------------|------------------|----------------------------|
+| process.runtime.dotnet.**gc.committed** | GC Committed Bytes | `By`  | ObservableGauge | `Int64`    |                  |                            |
+| process.runtime.dotnet.**gc.heapsize**  |                    | `By`  | ObservableGauge | `Int64`    | gen              | gen0, gen1, gen2, loh, poh |
+
+- [GCMemoryInfo.TotalCommittedBytes](https://docs.microsoft.com/dotnet/api/system.gcmemoryinfo.totalcommittedbytes?view=net-6.0#system-gcmemoryinfo-totalcommittedbytes):
+Gets the total committed bytes of the managed heap.
+
+- [GC.GetGCMemoryInfo().GenerationInfo[i].SizeAfterBytes](https://docs.microsoft.com/dotnet/api/system.gcgenerationinfo):
+Represents the size in bytes of a generation on exit of the GC reported in GCMemoryInfo.
+
+## JIT Compiler related metrics
+
+The metrics in this section can be enabled by setting the
+`RuntimeMetricsOptions.IsJitEnabled` switch.
+
+These metrics are only available for NET6_0_OR_GREATER.
+
+| Name                                            | Description              | Units       | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------------|--------------------------|-------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**il.bytes.jitted**      | IL Bytes Jitted          | `By`        | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**methods.jitted.count** | Number of Methods Jitted | `{methods}` | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**time.in.jit**          | Time spent in JIT        | `ns`        | ObservableCounter | `Int64`   |                  |                  |
+
+[JitInfo.GetCompiledILBytes](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompiledilbytes?view=net-6.0#system-runtime-jitinfo-getcompiledilbytes(system-boolean)):
+Gets the number of bytes of intermediate language that have been compiled.
+The scope of this value is global.
+
+[JitInfo.GetCompiledMethodCount](https://docs.microsoft.com/dotnet/api/system.runtime.jitinfo.getcompiledmethodcount?view=net-6.0#system-runtime-jitinfo-getcompiledmethodcount(system-boolean)):
+Gets the number of methods that have been compiled.
+The scope of this value is global.
```

Comment:
Do we need to state `The scope of this value is global.` on all of these? Could probably just drop that since it seems obvious (IMO) or move it to the heading so we only state it once? 🤷